### ISSUE TITLE
Upgrading to Rails 5 from 5.0.0.rc2 gives Bundler "activesupport" error

### ISSUE THREAD

**Issue by marcamillion** [rails/rails#25706]
### Steps to reproduce

Run 5.0.0.rc2, with Ruby 2.3.0, Bundler 1.12.5 and attempt to run either `bundle update rails`, `bundle update` or `bundle install` after changing the Gemfile to `gem 'rails', '~> 5.0.0'` ofcourse.

### Expected behavior
It should update easily, along with all dependencies.

### Actual behavior

```
$ bundle update
Fetching gem metadata from https://rubygems.org/
Fetching version metadata from https://rubygems.org/
Fetching dependency metadata from https://rubygems.org/
Resolving dependencies....................................................................................
Bundler could not find compatible versions for gem "activesupport":
  In Gemfile:
    rails (~> 5.0.0) was resolved to 5.0.0, which depends on
      activejob (= 5.0.0) was resolved to 5.0.0, which depends on
        activesupport (= 5.0.0)

    rails (~> 5.0.0) was resolved to 5.0.0, which depends on
      activejob (= 5.0.0) was resolved to 5.0.0, which depends on
        activesupport (= 5.0.0)

    rails (~> 5.0.0) was resolved to 5.0.0, which depends on
      activejob (= 5.0.0) was resolved to 5.0.0, which depends on
        activesupport (= 5.0.0)

    rails (~> 5.0.0) was resolved to 5.0.0, which depends on
      activejob (= 5.0.0) was resolved to 5.0.0, which depends on
        activesupport (= 5.0.0)

    rails (~> 5.0.0) was resolved to 5.0.0, which depends on
      activejob (= 5.0.0) was resolved to 5.0.0, which depends on
        activesupport (= 5.0.0)

    rails (~> 5.0.0) was resolved to 5.0.0, which depends on
      activejob (= 5.0.0) was resolved to 5.0.0, which depends on
        activesupport (= 5.0.0)

    rails (~> 5.0.0) was resolved to 5.0.0, which depends on
      activejob (= 5.0.0) was resolved to 5.0.0, which depends on
        activesupport (= 5.0.0)

    carrierwave (~> 0.11.2) was resolved to 0.11.2, which depends on
      activesupport (>= 3.2.0)

    rails (~> 5.0.0) was resolved to 5.0.0, which depends on
      activejob (= 5.0.0) was resolved to 5.0.0, which depends on
        globalid (>= 0.3.6) was resolved to 0.3.6, which depends on
          activesupport (>= 4.1.0)

    google-webfonts-rails (~> 0.0.4) was resolved to 0.0.4, which depends on
      activesupport (< 5.0, > 3.2.0)

    rails (~> 5.0.0) was resolved to 5.0.0, which depends on
      actioncable (= 5.0.0) was resolved to 5.0.0, which depends on
        actionpack (= 5.0.0) was resolved to 5.0.0, which depends on
          rails-dom-testing (~> 2.0) was resolved to 2.0.1, which depends on
            activesupport (< 6.0, >= 4.2.0)

    rspec-rails (~> 3.5.0.beta3) was resolved to 3.5.0, which depends on
      activesupport (>= 3.0)

```

Or `bundle install`:

```
$ bundle install
Fetching gem metadata from https://rubygems.org/
Fetching version metadata from https://rubygems.org/
Fetching dependency metadata from https://rubygems.org/
Resolving dependencies....................................................................................
Bundler could not find compatible versions for gem "activesupport":
  In Gemfile:
    rails (~> 5.0.0) was resolved to 5.0.0, which depends on
      activejob (= 5.0.0) was resolved to 5.0.0, which depends on
        activesupport (= 5.0.0)

    rails (~> 5.0.0) was resolved to 5.0.0, which depends on
      activejob (= 5.0.0) was resolved to 5.0.0, which depends on
        activesupport (= 5.0.0)

    rails (~> 5.0.0) was resolved to 5.0.0, which depends on
      activejob (= 5.0.0) was resolved to 5.0.0, which depends on
        activesupport (= 5.0.0)

    rails (~> 5.0.0) was resolved to 5.0.0, which depends on
      activejob (= 5.0.0) was resolved to 5.0.0, which depends on
        activesupport (= 5.0.0)

    rails (~> 5.0.0) was resolved to 5.0.0, which depends on
      activejob (= 5.0.0) was resolved to 5.0.0, which depends on
        activesupport (= 5.0.0)

    rails (~> 5.0.0) was resolved to 5.0.0, which depends on
      activejob (= 5.0.0) was resolved to 5.0.0, which depends on
        activesupport (= 5.0.0)

    rails (~> 5.0.0) was resolved to 5.0.0, which depends on
      activejob (= 5.0.0) was resolved to 5.0.0, which depends on
        activesupport (= 5.0.0)

    carrierwave (~> 0.11.2) was resolved to 0.11.2, which depends on
      activesupport (>= 3.2.0)

    rails (~> 5.0.0) was resolved to 5.0.0, which depends on
      activejob (= 5.0.0) was resolved to 5.0.0, which depends on
        globalid (>= 0.3.6) was resolved to 0.3.6, which depends on
          activesupport (>= 4.1.0)

    google-webfonts-rails (~> 0.0.4) was resolved to 0.0.4, which depends on
      activesupport (< 5.0, > 3.2.0)

    rails (~> 5.0.0) was resolved to 5.0.0, which depends on
      actioncable (= 5.0.0) was resolved to 5.0.0, which depends on
        actionpack (= 5.0.0) was resolved to 5.0.0, which depends on
          rails-dom-testing (~> 2.0) was resolved to 2.0.1, which depends on
            activesupport (< 6.0, >= 4.2.0)

    rspec-rails (~> 3.5.0.beta3) was resolved to 3.5.0, which depends on
      activesupport (>= 3.0)

```

Or `bundle update rails`
```

$ bundle update rails
This Bundle hasn't been installed yet. Run `bundle install` to update and install the bundled gems.
```

I believe I get the above message because I have deleted my `Gemfile.lock` hoping that bundler would rebuild it. I am yet to be able to do that though.

### System configuration
Here is my Gemfile:

```
source 'https://rubygems.org'
ruby "2.3.0"

gem 'rails', '~> 5.0.0'
gem 'puma', '~> 3.0'
gem 'bootstrap-sass'
gem 'sass-rails', '~> 5.0'
gem 'font-awesome-rails'
gem 'google-webfonts-rails', '~> 0.0.4'
gem 'uglifier', '>= 1.3.0'
gem 'coffee-rails', '~> 4.2'
gem 'jquery-rails'
gem 'jquery-ui-rails', '~> 5.0', '>= 5.0.5'
gem 'turbolinks', '~> 5'
gem 'jbuilder', '~> 2.0'
gem 'cancancan', '~> 1.14.0'
gem "devise", '~> 4.0.0.rc1'
gem 'pg'
gem 'rolify'
gem 'simple_form'
gem 'acts_as_votable'
gem 'roadie-rails'
gem 'rouge'
gem 'redcarpet'
gem 'friendly_id', '~> 5.1'
gem 'country_select', '~> 2.5', '>= 2.5.2'
gem 'summernote-rails', '~> 0.8.1.1'
gem 'best_in_place', '~> 3.1'
gem 'stripe', '~> 1.43', '>= 1.43.1'
gem 'carrierwave', '~> 0.11.2'
gem 'fog-aws', '~> 0.9.3'
gem 'twitter', '~> 5.16'
gem 'omniauth-twitter', '~> 1.2', '>= 1.2.1'

group :development, :test do
  gem 'letter_opener'
  gem 'rspec-rails', '~> 3.5.0.beta3'
  gem 'factory_girl_rails'
  gem 'shoulda'
  gem 'shoulda-matchers'
  gem 'capybara'
  gem 'database_cleaner'
  gem 'faker'
  gem 'launchy'
  gem 'guard-rails'
  gem 'guard-rspec', require: false
  gem 'pry-rails'
  gem 'pry-rescue'
  gem 'pry-stack_explorer'
end

group :development do
  gem 'web-console'
  gem 'listen', '~> 3.0.5'
  gem 'spring'
  gem 'spring-watcher-listen', '~> 2.0.0'
  gem 'better_errors'
  gem 'binding_of_caller'
  gem 'guard-bundler'
  gem 'annotate'
  gem 'rb-fchange', require: false
  gem 'rb-fsevent', require: false
  gem 'rb-inotify', require: false
  gem 'rails-erd'
  gem 'flamegraph'
  gem 'stackprof', '~> 0.2.7'
  gem 'rack-mini-profiler'
end

# Windows does not include zoneinfo files, so bundle the tzinfo-data gem
gem 'tzinfo-data', platforms: [:mingw, :mswin, :x64_mingw, :jruby]

group :production do
  gem 'rails_12factor'
end

```

**Comment by maclover7** [comment]
Looks like `google-webfonts-rails` is the issue:

```
google-webfonts-rails (~> 0.0.4) was resolved to 0.0.4, which depends on
      activesupport (< 5.0, > 3.2.0)
```

Closing as a third party issue 😬

**Comment by tke578** [comment]
Another ref https://github.com/thoughtbot/shoulda-matchers/issues/953
